src/ask.js

```diff
@@ -2,7 +2,8 @@ exports.handler = (event, context, callback) => {
   let axios = require('axios')
   let qs = require('querystring')
 
-  let { text = 'babel eslint error' } = qs.parse(event.body)
+  //by default its set to null
+  let { text = null } = qs.parse(event.body)
```

Comment:
I think we might want to catch empty string here as well.

How about just:

```
let { text } = qs.parse(event.body)
```

then later we can just check for truthiness `if (text) ..`

Reply:
actually we should probably call `trim()` as well

```
let text = qs.parse(event.body).text.trim()
```

Reply:
seems like `qs.parse(event.body).text.trim()` when empty spits out undefined
so I'm gonna do this like this `qs.parse(event.body.trim())`